core/runtime/src/main/java/io/quarkiverse/mcp/server/InitialRequest.java

```diff
@@ -9,9 +9,10 @@
  * @param protocolVersion the protocol version supported by the client (must not be {@code null})
  * @param clientCapabilities the capabilities supported by the client (must not be {@code null})
  * @param transport the transport used by the client (must not be {@code null})
+ * @param instructions the instructions used by the client
  */
 public record InitialRequest(Implementation implementation, String protocolVersion,
-        List<ClientCapability> clientCapabilities, Transport transport) {
+        List<ClientCapability> clientCapabilities, Transport transport, String instructions) {
```

Comment:
AFAIK the "initialize" request from the client does not contain instructions...

Reply:
MMhh indeed... fixed